notebooks/train_model.ipynb

```diff
@@ -147,67 +161,15 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 33,
-   "id": "0daef8c5-a0df-43bb-9c26-1e7f12928d22",
-   "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "\u001b[1m1/1\u001b[0m \u001b[32m━━━━━━━━━━━━━━━━━━━━\u001b[0m\u001b[37m\u001b[0m \u001b[1m0s\u001b[0m 177ms/step\n",
-      "The image is predicted to be normal.\n"
-     ]
-    }
-   ],
-   "source": [
-    "import numpy as np\n",
-    "from tensorflow.keras.preprocessing import image\n",
-    "import cv2\n",
-    "\n",
-    "# Load and preprocess an image\n",
-    "img_path = 'data/myimage/image_1.png'\n",
-    "\n",
-    "# Load the image using OpenCV\n",
-    "img = cv2.imread(img_path)\n",
-    "\n",
-    "# Resize the image to the required input size of the model (224x224) and convert to grayscale\n",
-    "img = cv2.resize(img, (224, 224))\n",
-    "gray_img = cv2.cvtColor(img, cv2.COLOR_BGR2GRAY)\n",
-    "\n",
-    "# Convert grayscale image to float32\n",
-    "gray_img = gray_img.astype('float32')\n",
-    "\n",
-    "# Normalize the image\n",
-    "normalized_img = gray_img / 255.0\n",
-    "\n",
-    "# Convert grayscale image to three-channel (RGB) image\n",
-    "rgb_img = cv2.cvtColor(normalized_img, cv2.COLOR_GRAY2RGB)\n",
-    "\n",
-    "# Expand dimensions to match the input shape expected by the model\n",
-    "x = np.expand_dims(rgb_img, axis=0)\n",
-    "\n",
-    "# Predict\n",
-    "prediction = model.predict(x)\n",
-    "\n",
-    "# Visualize result\n",
-    "if prediction < 0.5:\n",
-    "    print(\"The image is predicted to be normal.\")\n",
-    "else:\n",
-    "    print(\"The image is predicted to have cataract.\")\n"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 34,
+   "execution_count": 8,
    "id": "c0b41ae5-ca44-427b-a781-8947c63308d0",
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "\u001b[1m1/1\u001b[0m \u001b[32m━━━━━━━━━━━━━━━━━━━━\u001b[0m\u001b[37m\u001b[0m \u001b[1m0s\u001b[0m 200ms/step\n",
+      "\u001b[1m1/1\u001b[0m \u001b[32m━━━━━━━━━━━━━━━━━━━━\u001b[0m\u001b[37m\u001b[0m \u001b[1m2s\u001b[0m 2s/step\n",
```

Comment:
seems to successfully predict the sample image, but we need a larger and diverse set of test images  to assess the model's real-world performance.